tutorials/quickstart/tutorial.md

```diff
@@ -0,0 +1,162 @@
+---
+title: "NServiceBus Quick Start"
+reviewed: 2017-06-05
+summary: In this tutorial, we'll see why software systems built on asynchronous messaging using NServiceBus are superior to traditional synchronous HTTP-based web services.
+extensions:
+- !!tutorial
+  downloadAtTop: true
+---
+
+In this tutorial, we'll see why software systems built on asynchronous messaging using NServiceBus are superior to traditional synchronous HTTP-based web services. We'll also show how NServiceBus guarantees reliability and extensibility that can't be achieved with REST.
+
+This tutorial skips over some concepts and implementation details in order to get up and running quickly. If you'd prefer a more in-depth tutorial, check out [Introduction to NServiceBus](/tutorials/intro-to-nservicebus/). It will teach you the NServiceBus API and important concepts you need to learn to build successful message-based software systems.
```

Comment:
> check out Introduction to NServiceBus]

better with an "our" or "the" before "Introduction":

> check out our Introduction to NServiceBus

Did you also want to include "tutorial" at the end of that like you did at the end of this post?